src/documentation/components/ThemeSwitcher.vue

```diff
@@ -73,6 +73,21 @@
           src="/themes/lumen/images/favicon.svg"
         />
         <div class="-theme-name">Lumen Enterprise Portal</div></a
+      ><a
```

Comment:
@jllr Not related to the scope of this PR, but we will need to generate these elements from an array (in a separate PR) as the code is repeated for 4 themes now.